assets/models/system/distilroberta-base/model.yaml

```diff
@@ -1,6 +1,6 @@
 path:
   container_name: models
-  container_path: huggingface/distilroberta-base/1693401261/mlflow_model_folder
+  container_path: huggingface/distilroberta-base/1694425675/mlflow_model_folder
```

Comment:
Could you also share blobstorage save run link?

Reply:
Run link for saving to blob storage: https://ml.azure.com/runs/stoic_parcel_kyv6t43wdp?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#